actionmailbox/lib/generators/action_mailbox/install/install_generator.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module ActionMailbox
+  module Generators
+    class InstallGenerator < ::Rails::Generators::Base
+      source_root File.expand_path("../../../rails/generators/mailbox/templates", __dir__)
```

Comment:
Think this should just point to `"templates"`? Why can't we move them here like Action Text's install generator?

Reply:
I did that initially but later I realized the templates are also used in here: https://github.com/rails/rails/blob/master/actionmailbox/lib/rails/generators/mailbox/mailbox_generator.rb#L14-L16

Reply:
There's gotta be some other Rails generator that has hit the same thing, no? And found a better way. What's the precedent?

Reply:
Found `ScaffoldGenerator`(https://github.com/rails/rails/blob/master/railties/lib/rails/generators/css/scaffold/scaffold_generator.rb#L8) does require it in a better way.